llvm/test/tools/llvm-dwarfdump/dump_dwo.test

```diff
@@ -0,0 +1,121 @@
+## Check that llvm-dwarfdump works when dumping .o files with .dwo files.
+
+# RUN: yaml2obj %p/Inputs/dump_dwo.o.yaml -o %T/dump_dwo.o
+# RUN: yaml2obj %p/Inputs/dump_dwo.dwo.yaml -o %T/dump_dwo.dwo
+## We need to be in the same directory so we can find .dwo file
+# RUN: cd %T && llvm-dwarfdump --dwo %T/dump_dwo.o | FileCheck %s
+# RUN: cd %T && llvm-dwarfdump --name int --name char %T/dump_dwo.o | FileCheck %s --check-prefix=NAMES
+# RUN: cd %T && llvm-dwarfdump --lookup 0x10 %T/dump_dwo.o | FileCheck %s --check-prefix=LOOKUP
```

Comment:
Each run line doesn't need its own `cd` - can use one up-front. (maybe even before the `yaml2obj` lines, then those lines don't need to `%T/dump_dwo.o` they can write to `dump_dwo.o`, etc, unqualified.

Reply:
done